js/private/translate_pnpm_lock.js

```diff
@@ -104,26 +104,29 @@ async function main(argv) {
               },
           }
 
-    const rootImporters = importers['.']
+    const rootImporters = lockImporters['.']
     if (!rootImporters) {
         console.error('no root importers in lockfile')
         process.exit(1)
     }
 
-    const lockDependencies = {
-        ...(!prod && rootImporters.devDependencies
-            ? rootImporters.devDependencies
-            : {}),
-        ...(!dev && rootImporters.dependencies
-            ? rootImporters.dependencies
-            : {}),
-        ...(!noOptional && rootImporters.optionalDependencies
-            ? rootImporters.optionalDependencies
-            : {}),
-    }
-    const directDependencies = []
-    for (const name of Object.keys(lockDependencies)) {
-        directDependencies.push(pnpmName(name, lockDependencies[name]))
+    const importers = {}
```

Comment:
glad to see not much change here, I still want to drop the node dependency from our toolchain and port this back to starlark